maestro-ios/src/main/proto/idb.proto

```diff
@@ -870,4 +870,4 @@ message DapResponse {
 
 message SimulateMemoryWarningRequest {}
 
-message SimulateMemoryWarningResponse {}
\ No newline at end of file
+message SimulateMemoryWarningResponse {}
```

Comment:
Is something extra changed here?

Reply:
A newline was added at the end of the file, my IntellIJ does that when I open the file